core/src/codechain_machine.rs

```diff
@@ -46,22 +47,51 @@ impl CodeChainMachine {
         CodeChainMachine {
             params: vec![Params {
```

Comment:
Should we read the params from the Metadata DB?

Reply:
I removed this because I made the states have the params.